_data/footer.yml

```diff
@@ -8,7 +8,7 @@
     - title: Overviews/Guides
       url: "/overviews"
     - title: Language Specification
-      url: "http://scala-lang.org/files/archive/spec/2.12/"
+      url: "http://scala-lang.org/files/archive/spec/2.13/"
```

Comment:
Let's make it https.

Reply:
this is just one of many such http links, so instead of fixing just this one, I've opened a ticket suggesting it be fixed site-wide: https://github.com/scala/docs.scala-lang/issues/1459